src/types.ts

```diff
@@ -61,6 +60,11 @@ export interface QueryModel extends DataQuery {
   };
 }
 
+export enum EditorMode {
```

Comment:
Should we export this from grafana ui?

Reply:
Yeah think it should be. @scottlepp if you're still working on this, can you make sure [EditorMode](https://github.com/grafana/grafana/blob/main/public/app/features/plugins/sql/types.ts#L52) is being exported?